pset-iterators.js

```diff
@@ -183,6 +226,21 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 7:')
 
 // Add your code below this line
+const shortTermPresidents = presidents.filter(function (president) {
+  // today's president is special case but has not yet left office
+  if (president.left_office === null) {
+    president.left_office="2019-09-30"
+  }
+  let started=president.took_office.split("-")
+  let ended=president.left_office.split("-")
+  // including number of months in the calculation (did not include days.....)
+  let duration=parseInt(ended[0])-parseInt(started[0]) + (parseInt(ended[1])-parseInt(started[1]))/12
+  return duration < 4
+})
+
+console.log("Presidents who lasted less than four years in office (also individuals just short of 4 months):")
+console.log(shortTermPresidents)
```

Comment:
@AtleLohrmann Good Job here with this solution to return the presidents who served for less than 4 years.
I have some suggestions here
- For presidents who left office equals null instead of setting a current date you can just return false so you can lease those presidents of the list.
- For this example, nice work with the month calculation but for this example only looking up the year would've been sufficient.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L201)